mira/dkg/client.py

```diff
@@ -472,6 +470,22 @@ def get_transitive_closure(self, rels: Optional[List[str]] = None) -> Set[Tuple[
         return transitive_closure
 
 
+# Follows example here:
+# https://neo4j.com/docs/python-manual/current/session-api/#python-driver-simple-transaction-fn
+# and from the docstring of neo4j.Session.read_transaction
+@unit_of_work()
+def do_cypher_tx(
+        tx: Transaction,
+        query: str,
+        query_params: Optional[Dict] = None
+) -> List[List]:
+    result = tx.run(query, parameters=query_params)
+    values = []
+    for record in result:
+        values.append(record.values())
```

Comment:
Should this be simplified to a list comprehension?

Reply:
Tested and resolved in https://github.com/indralab/mira/pull/74/commits/bfa7aa89b91bc83daa6194473a45a1f25e3cd7e0